website/static/js/logTextParser.js

```diff
@@ -25,6 +25,12 @@ function ravenMessage (message, logObject) {
     }
 }
 
+function toRelativeUrl(url) {
+    var parser = document.createElement('a');
```

Comment:
I'm not sure; depending on what gets logged if you add an external link with the forward addon, that could be. It would probably be wise to verify that the window's url matches the link url before making it relative.